### DISCUSSION TITLE
Clarity works with what Angular version?

### DISCUSSION THREAD

**Discussion by danrald** [vmware/clarity#6261] [Q&A]
I created a new Angular project, 12, just to play around with clarity. Either I am doing something wrong or it does not work with angular 12. What version of Angular would be a good fit for Clarity? Clarity looks really nice, I want to make an app with it.
Thanks,
Daniel

**Comment by danrald** [comment]
Ok, I looked at the repo and the package.json has v12. I must be doing something work. I'll look further. Thanks

**Reply by Jinnie** [reply]
What you did is right - the version is in the dependencies in package.json.
We've started aligning the versions of Clarity Angular with Angular versions, so we can make it easy to find the correct Angular.
My suggestion is to check the Get Started instructions https://angular.clarity.design/documentation/get-started
